test/index.html

```diff
@@ -22,11 +22,12 @@
 <body id="body">
 <h1 id="qunit-header">
 	<a href="http://jqueryvalidation.org/">jQuery Validation Plugin</a> Test Suite
-	<a href="?jquery=1.7.2">jQuery 1.7.2</a>
-	<a href="?jquery=1.8.3">jQuery 1.8.3</a>
-	<a href="?jquery=1.9.1">jQuery 1.9.1</a>
-	<a href="?jquery=1.11.1">jQuery 1.11.1</a>
-	<a href="?jquery=git">jQuery Latest (git)</a>
+	<a href="?jquery=1.7.2&jquery.mockjax=1.5.3">jQuery 1.7.2</a>
```

Comment:
It's used in jquery-validation/lib/jquery.mockjax.js. File loaded by the test suite in order to choose the version of jquery.mockjax to use, depending on jquery version.